### DISCUSSION TITLE
Welcome to haunted Discussions!

### DISCUSSION THREAD

**Discussion by matthewp** [matthewp/haunted#387] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bennypowers** [comment]
Nice going on all the maintenance work today 💪

**Comment by BeefOnWeck** [comment]
Totally agree; it's nice to see the activity.  I discovered haunted a few months ago and decided to switch a hobby project over from Vue.  Feels like a breath of fresh air and I absolutely love being able to use hooks with custom web components.

**Comment by shroy** [comment]
I just wanted to say how much I appreciate this library! It has been so helpful in my projects. I was wondering, though, is it still being maintained?

**Reply by bennypowers** [reply]
It's not abandoned but it's not receiving frequent patches. Neither maintainer is an active user (correct me if I'm wrong @matthewp)

If any of the active users wanted to step up their contributions, I'd welcome it. I think the library occupies an important niche.
